dev-libs/opencl_headers/opencl_headers-2.2~20180318.recipe

```diff
@@ -9,26 +9,27 @@ tools, vision processing, and neural network training and inferencing."
 HOMEPAGE="https://www.khronos.org/opencl_headers/"
 COPYRIGHT="2008-2015 The Khronos Group Inc."
 LICENSE="MIT"
-REVISION="1"
+REVISION="2"
```

Comment:
For this recipe, `opencl_headers-2.2~20180318.recipe`, you don't need to bump `REVISION`.
(Feel free to ignore this comment if you wish to force a rebuild, of course :)